### PR TITLE
Cnp

### PR DESCRIPTION
**Pull request by mthava**
# Description

test build

### CODE REVIEW

File: src/test/java/uk/gov/hmcts/reform/divorce/transformservice/service/DivorceToCcdTransformationServiceTest.java [hunk 1]
```diff
@@ -1,72 +1,44 @@
...
-    private DivorceCaseToCCDMapper divorceCaseToCCDMapper;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.mockito.Mockito.mock;
 
-    @InjectMocks
+@RunWith(SpringRunner.class)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@ContextConfiguration(classes = CaseProgressionApplication.class)
+public class DivorceToCcdTransformationServiceTest {
```
Comment (mthava) [inline src/test/java/uk/gov/hmcts/reform/divorce/transformservice/service/DivorceToCcdTransformationServiceTest.java]: check why spring boot test is here

File: src/test/java/uk/gov/hmcts/reform/divorce/transformservice/service/PdfGeneratorServiceTest.java [hunk 2]
```diff
@@ -34,39 +34,39 @@ public void pdfGeneratorReturnAPdf() {
...
+        when(pdfGeneratorClient.generatePdf(submittedCase, "test")).thenReturn(pdfFile);
 
-        PdfFile pdfFileGenerated = pdfGeneratorService.generatePdf(submittedCase);
+        PdfFile pdfFileGenerated = pdfGeneratorService.generatePdf(submittedCase, "test");
         assertThat(pdfFileGenerated).isEqualTo(pdfFile);
         assertThat(pdfFileGenerated.getUrl()).isEqualTo("oneUrl");
         assertThat(pdfFileGenerated.toString()).isEqualTo("PdfFile(url=oneUrl, fileName=d8petition990)");
 
-        verify(pdfGeneratorClient).generatePdf(submittedCase);
+        verify(pdfGeneratorClient).generatePdf(submittedCase, "test");
```
Comment (mthava) [inline src/test/java/uk/gov/hmcts/reform/divorce/transformservice/service/PdfGeneratorServiceTest.java]: move test to variable

File: build.gradle [hunk 3]
```diff
@@ -0,0 +1,244 @@
...
+    id 'org.owasp.dependencycheck' version '3.1.2'
+    id 'org.sonarqube' version '2.6.2'
+    id 'com.github.ben-manes.versions' version '0.17.0'
+    id 'jacoco'
+    id 'pmd'
+    id 'checkstyle'
+}
+
+group = 'uk.gov.hmcts.divorce'
+version = '0.0.3'
```
Comment (dnenkov) [inline build.gradle]: Consider increasing the version - https://tools.hmcts.net/jira/browse/DIV-2537

File: automation-test/build.gradle [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+    maven { url "http://artifactory.reform.hmcts.net/artifactory/libs-release" }
+    jcenter()
+}
+
+gradle.startParameter.continueOnFailure=true
+
+check.dependsOn aggregate
+
+group = 'uk.gov.hmcts.reform.divorce'
+version = '0.0.3-SNAPSHOT'
```
Comment (dnenkov) [inline automation-test/build.gradle]: Consider increasing the version - https://tools.hmcts.net/jira/browse/DIV-2537

File: infrastructure/aat.tfvars [hunk 5]
```diff
@@ -0,0 +1,7 @@
+vault_env = "preprod"
+
+logging_level_org_springframework_web = "DEBUG"
+logging_level_uk_gov_hmcts_ccd = "DEBUG"
+draft_store_api_baseurl = "https://preproddraftstorelb.moneyclaim.reform.hmcts.net:4301"
```
Comment (dnenkov) [inline infrastructure/aat.tfvars]: Consider moving to draft store in CNP - https://tools.hmcts.net/jira/browse/DIV-2538

File: infrastructure/demo.tfvars [hunk 6]
```diff
@@ -0,0 +1,7 @@
+vault_env = "preprod"
+
+logging_level_org_springframework_web = "DEBUG"
+logging_level_uk_gov_hmcts_ccd = "DEBUG"
+draft_store_api_baseurl = "https://preproddraftstorelb.moneyclaim.reform.hmcts.net:4301"
```
Comment (dnenkov) [inline infrastructure/demo.tfvars]: Consider moving to draft store in CNP - https://tools.hmcts.net/jira/browse/DIV-2538

File: infrastructure/prod.tfvars [hunk 7]
```diff
@@ -0,0 +1,7 @@
+vault_env = "prod"
+
+logging_level_org_springframework_web = "INFO"
+logging_level_uk_gov_hmcts_ccd = "INFO"
+draft_store_api_baseurl = "https://proddraftstorelb.moneyclaim.reform.hmcts.net:4301"
```
Comment (dnenkov) [inline infrastructure/prod.tfvars]: Consider moving to draft store in CNP - https://tools.hmcts.net/jira/browse/DIV-2538

File: infrastructure/saat.tfvars [hunk 8]
```diff
@@ -0,0 +1,7 @@
+vault_env = "test"
+
+logging_level_org_springframework_web = "DEBUG"
+logging_level_uk_gov_hmcts_ccd = "DEBUG"
+draft_store_api_baseurl = "https://testdraftstorelb.moneyclaim.reform.hmcts.net:4302"
```
Comment (dnenkov) [inline infrastructure/saat.tfvars]: Consider moving to draft store in CNP - https://tools.hmcts.net/jira/browse/DIV-2538

File: infrastructure/sandbox.tfvars [hunk 9]
```diff
@@ -0,0 +1,7 @@
+vault_env = "test"
+
+logging_level_org_springframework_web = "DEBUG"
+logging_level_uk_gov_hmcts_ccd = "DEBUG"
+draft_store_api_baseurl = "https://testdraftstorelb.moneyclaim.reform.hmcts.net:4302"
```
Comment (dnenkov) [inline infrastructure/sandbox.tfvars]: Consider moving to draft store in CNP - https://tools.hmcts.net/jira/browse/DIV-2538

**Comment by Rob-Doherty** [comment]
Happy to remove the 'request for changes' block based on Dan's comments

**Pull request merged**